guides/common/modules/proc_creating-custom-provisioning-snippets.adoc

```diff
@@ -0,0 +1,33 @@
+[id="Creating_Custom_Provisioning_Snippets_{context}"]
+= Creating Custom Provisioning Snippets
+
+A custom provisioning snippets allows you to execute custom code during host provisioning.
+You can run code before and/or after the provisioning process.
+
+.Prerequisite
+* Depending on your provisioning template, multiple custom snippet hooks exist which you can use to include custom provisioning snippets.
+Ensure that you check your provisioning template first to verify which custom snippet you can use.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Templates* and click *Create Template*.
+. In the *Name* field, enter a name for your custom provisioning snippet.
+The name has to start with the name of a provisioning template that supports including custom provisioning snippets:
+** Append ` custom pre` to the name of a provisioning template to run code before provisioning a host.
+** Append ` custom post` to the name of a provisioning template to run code after provisioning a host.
```

Comment:
The introduction says you can run code before **and/or** after the provisioning, so is there an option for both before and after? Or do I just append both?

Reply:
Yes, you can use both if you want to, but with separate snippets.